src/datetime.js

```diff
@@ -1552,6 +1574,14 @@ export default class DateTime {
     return this.isValid ? this.ts : NaN;
   }
 
+  /**
+   * Returns the epoch seconds of this DateTime.
+   * @return {number}
+   */
+  toSeconds() {
+    return this.isValid ? Math.floor(this.ts / 1000) : NaN;
```

Comment:
Should we really be flooring here? Why not just return it with the decimals? (I think this really is a different case than the `toFormat` one)

Reply:
Yeah, I went back and forth on this one. If you think this is better as a float, I'm happy to change it.

Reply:
I amended my commit to not round seconds.